src/JustSaying/Fluent/TopicPublicationBuilder`1.cs

```diff
@@ -71,6 +77,43 @@ public TopicPublicationBuilder<T> WithWriteConfiguration(Action<SnsWriteConfigur
             return this;
         }
 
+        /// <summary>
+        /// Creates a tag with no value that will be assigned to the SNS topic.
+        /// </summary>
+        /// <param name="key">The key for the tag.</param>
+        /// <returns>
+        /// The current <see cref="TopicPublicationBuilder{T}"/>.
+        /// </returns>
+        /// <remarks>Tag keys are case-sensitive. A new tag with a key identical to that of an existing one will overwrite it.</remarks>
```

Comment:
The docs say the key is case sensitive, but you're using a case-insensitive comparer for the default `Tags` dictionary, meaning you could have only have the key "foo" though the docs imply you could have "foo", "Foo" and "fOo".

Reply:
🤦 good point. I changed the docs after realising they’re case sensitive but not the actual type declaration